### PR TITLE
FF128 PushMessageData.bytes() supported

### PR DESCRIPTION
**Pull request by hamishwillee**
FF128 adds support for `PushMessageData.bytes()` in https://bugzilla.mozilla.org/show_bug.cgi?id=1897871

This creates the associated docs page  and a release note.

Related docs work  can be tracked in #33990

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/128/index.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ This article provides information about the changes in Firefox 128 that affect d
 ### APIs
 
 - {{domxref('RTCRtpReceiver.getParameters()')}} and {{domxref('RTCRtpSender.getParameters()')}} are now supported, returning an object that describes the current codecs used for the encoding and transmission of media on the receiver and sender tracks, respectively. ([Firefox bug 1534687](https://bugzil.la/1534687)).
+- {{domxref("PushMessageData.bytes()")}} is now supported for returning the data from a push message as an {{domxref("Uint8Array")}} object. ([Firefox bug 1897871](https://bugzil.la/1897871)).
```
Comment (hamishwillee) [inline files/en-us/mozilla/firefox/releases/128/index.md:47]:
```suggestion
- {{domxref("PushMessageData.bytes()")}} is now supported for returning the data from a push message as an array of bytes in a {{domxref("Uint8Array")}} object. ([Firefox bug 1897871](https://bugzil.la/1897871)).
```

**Review by bsmth — APPROVED** [review]
Looking good! Thank you

File: files/en-us/mozilla/firefox/releases/128/index.md [hunk 2]
```diff
@@ -44,6 +44,7 @@ This article provides information about the changes in Firefox 128 that affect d
 ### APIs
 
 - {{domxref('RTCRtpReceiver.getParameters()')}} and {{domxref('RTCRtpSender.getParameters()')}} are now supported, returning an object that describes the current codecs used for the encoding and transmission of media on the receiver and sender tracks, respectively. ([Firefox bug 1534687](https://bugzil.la/1534687)).
+- {{domxref("PushMessageData.bytes()")}} is now supported for returning the data from a push message as an array of bytes in a {{domxref("Uint8Array")}} object. ([Firefox bug 1897871](https://bugzil.la/1897871)).
```
Comment (hamishwillee) [inline files/en-us/mozilla/firefox/releases/128/index.md:47]:
```suggestion
- {{domxref("PushMessageData.bytes()")}} is now supported for returning the data from a push message as an array of bytes in a {{jsxref("Uint8Array")}} object. ([Firefox bug 1897871](https://bugzil.la/1897871)).
```

File: files/en-us/web/api/pushmessagedata/index.md [hunk 3]
```diff
@@ -23,6 +23,8 @@ None.
   - : Extracts the data as an {{jsxref("ArrayBuffer")}} object.
 - {{domxref("PushMessageData.blob()")}}
   - : Extracts the data as a {{domxref("Blob")}} object.
+- {{domxref("PushMessageData.bytes()")}}
+  - : Extracts the data as a {{domxref("Uint8Array")}} object.
```
Comment (hamishwillee) [inline files/en-us/web/api/pushmessagedata/index.md:27]:
```suggestion
  - : Extracts the data as a {{jsxref("Uint8Array")}} object.
```

**Pull request merged**